src/controllers/controlpickermenu.cpp

```diff
@@ -354,46 +354,53 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "ScrollUp",
+                       tr("Scroll Up"),
+                       tr("Equivalent to pressing the PAGE UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextPlaylist",
-                       tr("Switch To Next View"),
-                       tr("Switch to the next view (library, playlist..)"),
+    addPrefixedControl("[Library]", "ScrollDown",
+                       tr("Scroll Down"),
+                       tr("Equivalent to pressing the PAGE DOWN key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectPrevPlaylist",
-                       tr("Switch To Previous View"),
-                       tr("Switch to the previous view (library, playlist..)"),
+    addPrefixedControl("[Library]", "ScrollVertical",
+                       tr("Scroll up/down"),
+                       tr("Scroll vertically in either direction using a knob, as if pressing PGUP/PGDOWN keys"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectTrackKnob",
-                       tr("Scroll To Next/Previous Track"),
-                       tr("Scroll up or down in library/playlist"),
+    addPrefixedControl("[Library]", "MoveLeft",
+                       tr("Move left"),
+                       tr("Equivalent to pressing the LEFT key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextTrack",
-                       tr("Scroll To Next Track"),
-                       tr("Scroll to next track in library/playlist"),
+    addPrefixedControl("[Library]", "MoveRight",
+                       tr("Move right"),
+                       tr("Equivalent to pressing the RIGHT key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectPrevTrack",
-                       tr("Scroll To Previous Track"),
-                       tr("Scroll to previous track in library/playlist"),
+    addPrefixedControl("[Library]", "MoveHorizontal",
+                       tr("Move left/right"),
+                       tr("Move horizontall in either direction using a knob, as if pressing LEFT/RIGHT keys"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "LoadSelectedIntoFirstStopped",
-                       tr("Load Track Into Stopped Deck"),
-                       tr("Load selected track into first stopped deck"),
+    addPrefixedControl("[Library]", "MoveFocusForward",
+                       tr("Move focus to right pane"),
+                       tr("Equivalent to pressing the TAB key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "AutoDjAddBottom",
```

Comment:
You can use ControlDoublePrivate::insertAlias to track them on old and new name.